sdk/js/src/service/applications.js

```diff
@@ -146,6 +148,21 @@ class Applications {
 
     return Marshaler.unwrapRights(result)
   }
+
+  // Events Stream
+
+  async openStream (identifiers, tail, after) {
+    const eventsUrl = `${this._stackConfig.is}/events`
```

Comment:
I guess you'll want these from the application server instead of the identity server